table.html

```diff
@@ -8,7 +8,7 @@
 <body>
 
 <table>
-    <caption>Tabulka kontaktů</caption>
+    <caption vlastniAtribut="pozadi">Tabulka kontaktů</caption>
```

Comment:
super si to vyzkouset :) Jen i kdyz to funguje, melo byt to byt spise jedno z nasledujicich 
```
<caption data-vlastni-atribut="pozadi">Tabulka kontaktů</caption>
<caption data-vlastniatribut="pozadi">Tabulka kontaktů</caption>
```

tj. zacinat `data-` a pak mala pismena a misto mezer typicky pomlcka